runtime/src/main/java/io/novafoundation/nova/runtime/extrinsic/ExtrinsicBuilderFactory.kt

```diff
@@ -55,7 +59,7 @@ class ExtrinsicBuilderFactory(
 
         val accountAddress = chain.addressOf(accountId)
 
-        val runtimeVersion = rpcCalls.getRuntimeVersion(chain.id)
+        val runtimeVersion = getRuntimeVersion(chain) ?: return emptySequence()
```

Comment:
Maybe we can call `rpcCalls.getRuntimeVersion(chain.id)` version here as a fallback instead of returning `emptySequence`

Reply:
Yeah thats a good idea. Lets, however, change request to be `state_getRuntimeVersion` instead of `chain_getRuntimeVersion`. We can create a separate request class similar to `RuntimeVersionRequest`